examples/express-ts/package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "clean": "rimraf example.db lib",
     "compile": "tsc",
+    "test": "mocha test.js",
```

Comment:
Not a merge blocker, but I think it'd be nice to automatically clean and compile before running the tests. If you add `npm-run-all` as a dev dependency, we can do this:

```json
    "pretest": "run-s clean compile"
```

(and while we're at it, change `start` to have a `prestart` instead of the `&&` which may not work on windows boxen under non-bash shells).

If you don't want to do this diff, I can do it after this PR merges. Your call.

Reply:
You'd need to build and run the server in a separate process too.

I'm no script whiz. You might want to do this to get things the way you want it.